callback_plugins/fail_if_no_hosts.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
These two callback should stay in validations-common which will be a dependency of validations-libs.